test/e2e/v1beta1/e2e_one_off_builds_test.go

```diff
@@ -69,6 +69,7 @@ var _ = Describe("Using One-Off Builds", func() {
 					ClusterBuildStrategy("buildpacks-v3").
 					Namespace(testBuild.Namespace).
 					Name(testID).
+					SourceType("Git").
 					SourceGit("https://github.com/shipwright-io/sample-go.git").
```

Comment:
I think it makes sense, that `SourceGit` also sets the type in this factory pattern.

Reply:
added